.github/workflows/build-pr.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           ktlint_version: 0.43.2
   enhetstester:
     name: Enhetstester
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest-16-cores
```

Comment:
Er ks-sak hvitelistet? https://nav-it.slack.com/archives/C60FFACN5/p1666805558366079?thread_ts=1666796307.951419&cid=C60FFACN5